packages/manager/src/constants.ts

```diff
@@ -275,3 +275,7 @@ export const FIREWALL_GET_STARTED_LINK =
   'https://www.linode.com/docs/products/networking/cloud-firewall/get-started/';
 export const FIREWALL_LIMITS_CONSIDERATIONS_LINK =
   'https://www.linode.com/docs/products/networking/cloud-firewall/#limits-and-considerations';
+
+// Sold Out plans
+export const PLAN_IS_SOLD_OUT_COPY =
+  'This plan has no availability for the selected region. Please select a smaller plan or the same plan in another region.';
```

Comment:
```suggestion
  'This plan has no availability for the selected region. Please select a different plan or the same plan in another region.';
```

Was "smaller" picked because all plan above a certain size will **always** be sold out? (is it possible for a Dedicated 32GB to be sold out but a Dedicated 64GB to be in stock)  

It feels weird to me to assume the user should pick a smaller plan.

Reply:
This was the recommended copy (and yes if it is based on the host's availability so theoretically a bigger plan would never be available), but i do prefer using `different` as well so I made that update 👍